_drafts/EH-tidyx.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Community Highlight: Ellis Hughes's TidyX Screencast!"
+author: Lauren Wolfe
+categories:
+  - Community
+  - Technical
+tags: 
+  - highlight
+  - tidyverse
+  - R
+---
+
+As we've been reassessing our work here at The Coop, we have found that there is a need to facilitate more interaction between the people that make up our community. It makes sense - we no longer have easy access to a quick catch up in the hallways or over lunch on campus and many of us have settled into our new socially-distant lifestyle. This is why The Coop Blog is going to shift some focus back onto our community by highlighting members and their data related work. For our first community highlight we're showcasing Ellis and his TidyX screencast!
+
+Ellis is a self-described bioengineer turned data scientist who works as a statistical programmer within the Statistical Center for HIV/AIDS Research and Prevention (SCHARP) here at Fred Hutch. Folks who've spent a lot of time around The Coop or the R community in general will likely already be familiar with him as he's an regular user of The Coop Slack channel and has hosted various Coop community groups such as the data visualizaion group. Outside of his work at Fred Hutch, Ellis is an active member of the larger R community contributing to open-source projects and organizing the local Seattle UseR group.
```

Comment:
include a link to SCHARP? https://www.fredhutch.org/en/research/divisions/vaccine-infectious-disease-division/research/biostatistics-bioinformatics-and-epidemiology/statistical-center-for-hiv-aids-research-and-prevention/about.html